lib/reverse_sentence.rb

```diff
@@ -1,4 +1,46 @@
-# A method to reverse the words in a sentence, in place.
+# A method to reverse the my_sentences in a sentence, in place.
+# Time: O(n^2)
```

Comment:
The time complexity of your algorithm is actually O(n). Take a read on an explanation on: https://github.com/Ada-C10/reverse_sentence/blob/solution/lib/reverse_sentence.rb

I'd also encourage you to elaborate your time and space complexity reasoning. Often, being able to explain your logic and reasoning is more important than getting the correct final answer.